wrappers/src/shared_realm_cs.cpp

```diff
@@ -754,6 +754,22 @@ REALM_EXPORT bool shared_realm_remove_type(const SharedRealm& realm, uint16_t* t
     });
 }
 
+REALM_EXPORT bool shared_realm_remove_all(const SharedRealm& realm, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        realm->verify_in_write();
+
+        realm::Group& all_tables = realm->read_group();
```

Comment:
```suggestion
        auto& group = realm->read_group();
```

`all_tables` is a bit of a weird name for the group - even though we're using it to get the tables, that is not its sole purpose, so let's go with the more boring, but accurate `group`.